package.json

```diff
@@ -3,6 +3,10 @@
   "version": "0.1.0",
   "description": "Sudoku is a logic-based, combinatorial number-placement puzzle.",
   "author": "Mark <mark@remarkablemark.org>",
```

Comment:
And since we're modifying this file, do you mind adding yourself as a contributor?